shell/common/rasterizer.h

```diff
@@ -232,8 +234,11 @@ class Rasterizer final : public SnapshotDelegate {
   ///
   /// @param[in]  pipeline  The layer tree pipeline to take the next layer tree
   ///                       to render from.
+  /// @param[in]  discardCallback if specified and returns true, the layer tree
+  //                              is discarded instead of being rendered
   ///
-  void Draw(fml::RefPtr<Pipeline<flutter::LayerTree>> pipeline);
+  void Draw(fml::RefPtr<Pipeline<flutter::LayerTree>> pipeline,
+             DiscardCallback discardCallback = nullptr);
```

Comment:
Rather than passing `nullptr` as the default here, what do you think of having a static callback that resolved to `false` each time?